### ISSUE TITLE
install issue

### ISSUE THREAD

**Issue by ssdr** [vikstrous/pirate-get#81]
hey, when i just press pip3 install pirate-get, i got this error:

![Uploading image.png…]()

hope for your reply, thx~

**Comment by ssdr** [comment]
![image](https://cloud.githubusercontent.com/assets/270285/12370432/968036dc-bc4b-11e5-8830-f2834116036b.png)

**Comment by vikstrous** [comment]
Sounds like you need to install libxml. Let me know if you figure out the name of the brew package you need and we'll document it.

**Comment by cesarpachon** [comment]
same problem here (ubuntu 15) sudo apt-get install libxml2 says already latest version

**Comment by vranki** [comment]
Same here on Ubuntu 16.04:

> pip3 install pirate-get

Install fails due to:

```
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Isrc/lxml/includes -I/usr/include/python3.5m -c src/lxml/lxml.etree.c -o build/temp.linux-x86_64-3.5/src/lxml/lxml.etree.o -w
    In file included from src/lxml/lxml.etree.c:320:0:
    src/lxml/includes/etree_defs.h:14:31: fatal error: libxml/xmlversion.h: No such file or directory
    compilation terminated.
    Compile failed: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    cc -I/usr/include/libxml2 -c /tmp/xmlXPathInitorbkzi3b.c -o tmp/xmlXPathInitorbkzi3b.o
    /tmp/xmlXPathInitorbkzi3b.c:1:26: fatal error: libxml/xpath.h: No such file or directory
    compilation terminated.
    *********************************************************************************
    Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?
    *********************************************************************************
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

```

```
$ sudo apt install libxml2
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libxml2 is already the newest version (2.9.3+dfsg1-1ubuntu0.1).
0 upgraded, 0 newly installed, 0 to remove and 19 not upgraded.
```

**Comment by vranki** [comment]
Installing libxml2-dev changes the error:

```
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Isrc/lxml/includes -I/usr/include/python3.5m -c src/lxml/lxml.etree.c -o build/temp.linux-x86_64-3.5/src/lxml/lxml.etree.o -w
    In file included from src/lxml/lxml.etree.c:320:0:
    src/lxml/includes/etree_defs.h:14:31: fatal error: libxml/xmlversion.h: No such file or directory
    compilation terminated.
```

**Comment by rnhmjoj** [comment]
@vikstrous libxml is a dependency of pyquery, right? Can't we switch to something not requiring a C library? All the recent issues were about this.

**Comment by vikstrous** [comment]
Yeah, I'm okay with switching. PRs welcome.

On June 23, 2016 6:41:13 AM PDT, Michele Guerini Rocco <notifications@github.com> wrote:
>@vikstrous libxml is a dependency of pyquery, right? Can't we switch to
>something not requiring a C library? All the recent issues were about
>this.
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/vikstrous/pirate-get/issues/81#issuecomment-228053234

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by vikstrous** [comment]
Looks like we would have to switch to beautiful soup because pyquery depends on lxml.

**Comment by rnhmjoj** [comment]
That's seems that best option. I'll work on it.

**Comment by rnhmjoj** [comment]
Fixed in 41b3d56721d6ae3e16ae97aa8bc066a5b6c5e77e
